src/main/java/it/gov/pagopa/payment/service/TransactionServiceImpl.java

```diff
@@ -26,15 +27,15 @@ public SyncTrxStatusDTO getTransaction(String id, String userId) {
         .orElse(null);
 
     if (transactionInProgress == null) {
-      throw new ClientExceptionNoBody(
-          HttpStatus.NOT_FOUND,
-          "NOT FOUND");
+      throw new TransactionNotFoundOrExpiredException(
+          ExceptionCode.TRX_NOT_FOUND_OR_EXPIRED,
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor